ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/LogWithMetadata.java

```diff
@@ -1,18 +1,20 @@
 /*
- * Copyright ConsenSys AG.
  *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
+ *  * Copyright ConsenSys AG.
```

Comment:
This looks like the header now has double multiplication sign `* *` ...is that right?

Reply:
IDE config mistake on my side.